src/Bicep.Core.Samples/Files/InvalidVariables_LF/main.diagnostics.bicep

```diff
@@ -132,9 +132,9 @@ var objWithInterp = {
 
 // invalid fully qualified function access
 var mySum = az.add(1,2)
-//@[15:18) [BCP107 (Error)] The function "add" does not exist in namespace "az". |add|
+//@[15:18) [BCP109 (Error)] The type "az" does not contain function "add". |add|
 var myConcat = sys.concat('a', az.concat('b', 'c'))
-//@[34:40) [BCP107 (Error)] The function "concat" does not exist in namespace "az". |concat|
+//@[34:40) [BCP109 (Error)] The type "az" does not contain function "concat". |concat|
```

Comment:
>The [](start = 28, length = 3)

Can we special case this and restore the original message? As a user unfamiliar with the internals, I would have no idea that "type az" means namespace.

Reply:
Done